### PR TITLE
ffmpeg: add version 4.4.3

### PR DESCRIPTION
**Pull request by kambala-decapitator**
Specify library name and version:  **ffmpeg/4.4.3**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

Tested locally:
- macOS Intel (native)
- iOS arm64 (cross) with `with_libx264` off as it results in build error (but is fixed in #13388)

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by kambala-decapitator** [comment]
I see there's #13235 for v2 changes. Or should I still fix v2 linter errors in this PR?

**Comment by MartinDelille** [comment]
> I see there's https://github.com/conan-io/conan-center-index/pull/13235 for v2 changes. Or should I still fix v2 linter errors in this PR?

No

File: recipes/ffmpeg/all/conanfile.py [hunk 1]
```diff
@@ -886,7 +892,7 @@ def package_info(self):
             if self.options.get_safe("with_coreimage"):
                 self.cpp_info.components["avfilter"].frameworks.append(
                     "CoreImage")
-            if tools.Version(self.version) >= "5.0" and tools.is_apple_os(self.settings.os):
+            if Version(self.version) >= "5.0" and tools.is_apple_os(self.settings.os):
```
Comment (jwillikers) [inline recipes/ffmpeg/all/conanfile.py:895]:
```suggestion
            if Version(self.version) >= "5.0" and is_apple_os(self):
```

File: recipes/ffmpeg/all/conanfile.py [hunk 2]
```diff
@@ -1,6 +1,9 @@
-from conan.tools.files import rename
-from conans import ConanFile, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
```
Comment (jwillikers) [inline recipes/ffmpeg/all/conanfile.py:3]:
```suggestion
from conan.tools.apple import is_apple_os
from conan.tools.build import cross_building
```

**Review by jwillikers — CHANGES_REQUESTED** [review]

**Review by jwillikers — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**